libs/auth/Makefile

```diff
@@ -0,0 +1,4 @@
+install-script-dependencies:
+	sudo apt install -y default-jdk wget
+	wget 'https://github.com/martinpaljak/GlobalPlatformPro/releases/download/v20.08.12/gp.jar'
```

Comment:
Good catch! Will open a PR for this

Reply:
@arsalansufi, I can move this into the build system. I'd like to avoid adding new install dependencies to Makefiles if possible. Let me know if you have questions or concerns, but I've opened the issue here: https://github.com/votingworks/vxsuite-build-system/issues/11

Reply:
Awesome, thank you!

Reply:
@amcmanus One more piece of context for you: The dependencies in this file (Java, `wget`, `gp.jar`) are actually just necessary for initial card configuration at the Bakery. The main dependencies necessary for actual production auth (`libpcsclite1`, `libpcsclite-dev`) were added in this commit a few PRs back: https://github.com/votingworks/vxsuite/pull/3037/commits/f0ba256494741c41cf7a7d2402a71e796798337c